Homework_3.py

```diff
@@ -0,0 +1,48 @@
+input_str='''homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+'''
+
+import re # library for useing regex
+
+# 1. Normalize the text
+
+normilized_str = '' #new empty string for normilized string
+
+for element in input_str.lower().split('.'): # circle for list of sentences and make all elements in lower case
```

Comment:
I suggest replacing `element` with more meaningful name like `sentence` to make code more readable

Reply:
Done